staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -198,7 +198,7 @@ func newAuthenticator(c *cache, config *api.ExecConfig, cluster *clientauthentic
 
 		stdin:       os.Stdin,
 		stderr:      os.Stderr,
-		interactive: term.IsTerminal(int(os.Stdout.Fd())),
+		interactive: term.IsTerminal(int(os.Stdin.Fd())),
```

Comment:
I wonder if the rationale here was that unless stdout was visible to the user, the exec plugin couldn't reasonably prompt for input?

Reply:
talked with @enj, who pointed out that even if that were the case, this check wouldn't make sense, since the plugins prompted on stderr